quirks/password-rules.json

```diff
@@ -92,6 +92,9 @@
     "comcastpaymentcenter.com": {
         "password-rules": "minlength: 8; maxlength: 20; required: lower, upper; required: digit; max-consecutive: 2;"
     },
+    "comdirect.de": {
+        "password-rules": "minlength: 6; maxlength: 6; required: digit; digit; digit; digit; digit; digit; "
+    },
```

Comment:
Looks good, I am also a customer there and other variations are not possible.

Reply:
But I noticed a little something, the website seems not to build the rules with numbers correctly, required: digit; digit; digit; digit; digit; digit; is not valid required: digit; would be correct